Add option to specify a0 instead of e_max for the laser (#1770)
ECP-WarpX/WarpX: Docs/source/running_cpp/parameters.rst

    .. math::

        E_{max} = a_0 \frac{2 \pi m_e c^2}{e\lambda} = a_0 \times (4.0 \cdot 10^{12} \;V.m^{-1})

    When running a **boosted-frame simulation**, provide the value of ``<laser_name>.e_max``
    in the laboratory frame, and use ``warpx.gamma_boost`` to automatically
    perform the conversion to the boosted frame.

* ``<laser_name>.a0`` (`float` ; dimensionless)
    Peak normalized amplitude of the laser field (given in the lab frame, just as ``e_max`` above).
    See the description of ``<laser_name>.e_max`` for the conversion between ``a0`` and ``e_max``.

>     One of ``a0`` and ``e_max`` must be specified. When both are specified, ``a0`` is being used.

Author: ```suggestion
    Exactly one of ``a0`` and ``e_max`` must be specified.
```

    .. math::

        E_{max} = a_0 \frac{2 \pi m_e c^2}{e\lambda} = a_0 \times (4.0 \cdot 10^{12} \;V.m^{-1})

    When running a **boosted-frame simulation**, provide the value of ``<laser_name>.e_max``
    in the laboratory frame, and use ``warpx.gamma_boost`` to automatically
    perform the conversion to the boosted frame.

* ``<laser_name>.a0`` (`float` ; dimensionless)
    Peak normalized amplitude of the laser field (given in the lab frame, just as ``e_max`` above).

>     For a laser with a wavelength :math:`\lambda = 0.8\,\mu m`, the normalized amplitude

Reviewer: Just a bit confusing: here we mention a specific central wavelength but below in the formula we express it as a general expression.

Author: Right, so I just referred to the `e_max` description here, to avoid confusion and duplication.
